IDEAS/Media/Specialized/DryAir/package.mo

```diff
@@ -0,0 +1,73 @@
+within IDEAS.Media.Specialized;
+package DryAir "Package with moist air model that decouples pressure and temperature"
+  extends IDEAS.Media.Specialized.BaseClasses.PartialSimpleMedium(
+     mediumName="Air",
```

Comment:
check whether values are the same as in the Buildings library

Reply:
there were indeed some errors here. See also https://github.com/Mathadon/modelica_MPC/commit/827601ca01b47bb6422383fd4568148e5d3718c8